openassessment/xblock/static/sass/oa/views/_oa-base.scss

```diff
@@ -1086,7 +1086,7 @@
     }
 
     .action--save {
-      @extend %action-button
+      @extend %action-button !optional
```

Comment:
It was failing quality builds, not sure why it hadn't been before. I'll double-check to ensure that's still the case.